resources/views/frontend/search-courses/course-list.blade.php

```diff
@@ -18,9 +18,12 @@
         <div class="row">
             <div class="col-md-12">
                 <div class="card mb-2">
-                    <div class="card-header">
-                        <h2 class="card-header-title text-center text-dark font-weight-bold">{{__('generic.courses')}}</h2>
+                    <div class="card-header px-5" style="border-bottom: none">
+                        <h2 class="card-header-title text-start mb-0 text-dark font-weight-bold" style="transform: translateY(20%)">
+                            {{__('generic.courses')}}
+                        </h2>
                     </div>
+                    <hr class="mt-2 mb-0" style="background-color: transparent; border: 1px solid rgba(0, 0, 0, 0.125);">
```

Comment:
` <hr class="mt-2 mb-0 mx-4" style="border: 0; height: 1px; background-image: linear-gradient(to right, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0));">`
Similarly, just copy the given for all the `<hr>` tags in `index.blade.php` and `skill-videos.blade.php`.